tests/fixtures/warehouse/organisation_clusters.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import pytest
+
+@pytest.fixture
+def organisation_cluster(conn):
+    cluster_ghent = {
+        'canonical': 'Ghent University Hospital',
+        'variations': ['Ghent University Hospital', 'Ghent University']
+    }
```

Comment:
These variations feel like should be in different clusters, as they look like different organisations.